crates/bevy_state/src/state/sub_states.rs

```diff
@@ -127,11 +125,10 @@ pub use bevy_state_macros::SubStates;
 ///     /// We then define the compute function, which takes in the [`Self::SourceStates`]
 ///     fn should_exist(sources: Option<AppState>) -> Option<Self> {
 ///         match sources {
-///             /// When we are in game, so we want a GamePhase state to exist, and the default is
-///             /// GamePhase::Setup
-///             Some(AppState::InGame { .. }) => Some(GamePhase::Setup),
-///             /// Otherwise, we don't want the `State<GamePhase>` resource to exist,
-///             /// so we return None.
+///             /// When we are in game, so we want a GamePhase state to exist.
```

Comment:
Writing error that was already here before this PR, but this should probably be "We are in game, so we want ..."

Reply:
```suggestion
///             /// When we are in game, we want a GamePhase state to exist.
```